### PR TITLE
add requirements.txt, note it in docs

### LINKED ISSUE #64: gln depends on dateutil
Might be worth documenting that you need to `pip install dateutil` for this command to work.

### PR DESCRIPTION
**Pull request by ryan-williams**
fixes #64

### CODE REVIEW

**Comment by ryan-williams** [comment]
@danvk lmk if this looks reasonable

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ These were written alongside other bash and python helpers that I may not have p
 * `source .git-rc` from your `.bashrc`. This will give you lots of helpful aliases!
     * You might want to make sure it's not clobbering ones you already have/use.
     * It will also put several relevant directories from this repo on your `$PATH`
+* `pip install -r requirements.txt`; currently just makes sure you have `dateutil` available, which some scripts here need.
```
Comment (danvk) [inline README.md]: Maybe note that you need to install this globally, rather than in any sort of virtualenv.

**Comment by danvk** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (ryan-williams) [inline README.md]: cool, done

**Pull request merged**